terraform/modules/hub/hub_saml_proxy.tf

```diff
@@ -96,6 +108,50 @@ module "saml_proxy" {
   image_name                 = "verify-saml-proxy"
 }
 
+module "saml_proxy_fargate" {
+  source = "./modules/ecs_fargate_app"
+
+  deployment = var.deployment
+  app        = "saml-proxy"
+  domain     = local.root_domain
+  vpc_id     = aws_vpc.hub.id
+  lb_subnets = aws_subnet.internal.*.id
+  task_definition = templatefile("${path.module}/files/tasks/hub-saml-proxy.json",
+    {
+      image_identifier                 = "${local.tools_account_ecr_url_prefix}-verify-saml-proxy@${var.hub_saml_proxy_image_digest}"
+      nginx_image_identifier           = local.nginx_image_identifier
+      domain                           = local.root_domain
+      optional_links                   = ""
+      deployment                       = var.deployment
+      location_blocks_base64           = base64encode(local.saml_proxy_fargate_location_blocks)
+      region                           = data.aws_region.region.id
+      account_id                       = data.aws_caller_identity.account.account_id
+      event_emitter_api_gateway_url    = var.event_emitter_api_gateway_url
+      rp_truststore_enabled            = var.rp_truststore_enabled
+      certificates_config_cache_expiry = var.certificates_config_cache_expiry
+      memory_hard_limit                = var.saml_proxy_memory_hard_limit
+      jvm_options                      = var.jvm_options
+      log_level                        = var.hub_saml_proxy_log_level
+  })
+  container_name    = "nginx"
+  container_port    = "8443"
+  number_of_tasks   = var.number_of_apps
+  health_check_path = "/service-status"
+  tools_account_id  = var.tools_account_id
+  image_name        = "verify-saml-proxy"
+  certificate_arn   = var.wildcard_cert_arn
+  ecs_cluster_id    = aws_ecs_cluster.fargate-ecs-cluster.id
+  cpu               = 2048
+  # for a CPU of 2048 we need to set a RAM value between 4096 and 16384 (inclusive) that is a multiple of 1024.
+  memory  = ceil(max(var.saml_proxy_memory_hard_limit + 250, 4096) / 1024) * 1024
+  subnets = aws_subnet.internal.*.id
+  additional_task_security_group_ids = [
+    aws_security_group.scraped_by_prometheus.id,
```

Comment:
```suggestion
```
This line is not needed.  The in-hindsight-badly-named `scraped_by_prometheus` security group opens port 9100 (node_exporter) and 9479 (beat-exporter), neither of which are relevant to fargate apps.

Reply:
(yes, we should remove it from config-v2 as well, I'll raise a PR for that)

Reply:
https://github.com/alphagov/verify-infrastructure/pull/483